README.md

```diff
@@ -1,70 +1,125 @@
+### Prerequisites
+In order to run this project you need: to dowload it or clone it to your machine.
 
-### Code Splitting
+### Setup
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
 
-### Analyzing the Bundle Size
+1- Clone the repository to your local machine using the following command:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+git clone https://github.com/mohamedgamaleldin999999/my-personal-portfolio
 
-### Making a Progressive Web App
+2- Change into the project directory:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+cd [project directory]
 
-### Advanced Configuration
+3- Install the required dependencies:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+npm install
 
-### Deployment
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
 
-### `npm run build` fails to minify
+## 👥 Authors <a name="authors"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+👤 **Mohamed-Gamaleldin**
+
+- GitHub: [@githubhandle](https://github.com/mohamedgamaleldin999999)
+- Twitter: [@twitterhandle](https://twitter.com/Mohamme43086002)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohammed-jamal-949366221/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+[Feature1] :Add Javascript
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, subscribe to my [Youtube Channel](https://www.youtube.com/channel/UCGFAz2PASu5cJT9aFYrI7fg)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my Microverse learning partners: Winner, Marie and Ismail.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Well done and congratulations to start your `front-end` development journey using the `CRA` template for `react` applications. It seems like you added the `LICENSE.txt` file but you gave the wrong path. Because of this your `LICENSE` link is broken. Please fix the issue by following the below suggestions.

```suggestion
This project is [MIT](./LICENSE.txt) licensed.
```

### Right now your `LICENSE` link results in
![Screenshot from 2023-08-10 22-56-06](https://github.com/mohamedgamaleldin999999/Math-Magicians/assets/66476329/3b62a446-ca2d-4dd1-afb0-eb41a354230a)